src/app/admin/admin-reports/filtered-items/filtered-items-model.ts

```diff
@@ -0,0 +1,23 @@
+import { Item } from "src/app/core/shared/item.model";
```

Comment:
This line has a lint failure (run `yarn lint` locally to see it). We require single quotes in all import statements. So, change this to:
```
import { Item } from 'src/app/core/shared/item.model';
```